common/character_interactions/wc_debug_interactions.txt

```diff
@@ -48,4 +48,63 @@ become_undead_interaction = {
 		scope:recipient = { become_undead_effect = yes }
 	}
 	auto_accept = yes
+}
+wc_change_elemental_goodwill_interaction = {
+	category = interaction_debug_main
+	use_diplomatic_range = no
+	ignores_pending_interaction_block = yes
+	
+	common_interaction = yes
+
+	is_shown = {
+		debug_only = yes
+	}
+
+	send_name = "Change Elemental Goodwill"
+
+	send_option = { flag = flag_fire localization = "Fire" }
+	send_option = { flag = flag_water localization = "Water" }
+	send_option = { flag = flag_air localization = "Air" }
+	send_option = { flag = flag_earth localization = "Earth" }
+	send_option = { flag = flag_remove localization = "Lower Elemental Goodwill instead of raising it" }
+
+	send_options_exclusive = no
+
+	on_accept = {
+		scope:recipient = {
+			if = {
+				limit = { always = scope:flag_fire }
+				if = {
+					limit = { always = scope:flag_remove }
+					wc_change_AMOUNT_ELEMENT_favour_effect = { ELEMENT = elemental_fire AMOUNT = -50 }
+				}
+				else = { wc_change_AMOUNT_ELEMENT_favour_effect = { ELEMENT = elemental_fire AMOUNT = 50 } }
+			}
+			if = {
+				limit = { always = scope:flag_water }
+				if = {
+					limit = { always = scope:flag_remove }
+					wc_change_AMOUNT_ELEMENT_favour_effect = { ELEMENT = elemental_water AMOUNT = -50 }
+				}
+				else = { wc_change_AMOUNT_ELEMENT_favour_effect = { ELEMENT = elemental_water AMOUNT = 50 } }
+			}
+			if = {
+				limit = { always = scope:flag_air }
+				if = {
+					limit = { always = scope:flag_remove }
+					wc_change_AMOUNT_ELEMENT_favour_effect = { ELEMENT = elemental_air AMOUNT = -50 }
+				}
+				else = { wc_change_AMOUNT_ELEMENT_favour_effect = { ELEMENT = elemental_air AMOUNT = 50 } }
+			}
+			if = {
+				limit = { always = scope:flag_earth }
+				if = {
+					limit = { always = scope:flag_remove }
```

Comment:
As far as I can see, both variations are used in character interactions

Tho, sure, I have no no problems with replacing it

<details>
<summary>Click to expand</summary>

![image](https://user-images.githubusercontent.com/46576860/155899358-0b8ee0cb-e554-46f5-8d15-8119ff944385.png)
![image](https://user-images.githubusercontent.com/46576860/155899394-50c05a44-b79a-4375-997e-c8090366ba3d.png)


</details>

Reply:
> As far as I can see, both variations are used in character interactions

It may be the same though we know how `exists` works.
It returns true if the right side exists. Otherwise, returns false.